### PR TITLE
docs: add keys section and scroll_exits option

### PR DESCRIPTION
**Pull request by tessus**
I wasn't able to add the version info, since I didn't know when this will be available: `18.0.2`, `18.0.3`, `18.1.0`, ...

Btw, I've also noticed that the `[sync]` section and the `records` option is missing from the docs.

### CODE REVIEW

**Comment by ellie** [comment]
> Btw, I've also noticed that the [sync] section and the records option is missing from the docs.

Presently intentional, I'll fill it in later! Still sorting out some of the implementation details, so would like to ensure the people using it read release notes/the forum (hence it's only detailed there)

File: src/content/docs/configuration/config.mdx [hunk 1]
```diff
@@ -423,3 +423,16 @@ common_prefix = [
 
 Configures commands that should be totally stripped from stats calculations. For example, 'sudo' should be ignored.
 
+## keys
+This section of client config is specifically for configuring key-related settings.
+
+```toml
+[stats]
```
Comment (ellie) [inline src/content/docs/configuration/config.mdx:430]:
```suggestion
[keys]
```

**Comment by ellie** [comment]
otherwise, would it be possible to get a rebase please? 🙏

File: src/content/docs/configuration/config.mdx (hunk 1 above)
Comment (tessus) [reply src/content/docs/configuration/config.mdx:430]: oops. thanks.

**Review by ellie — APPROVED** [review]

**Pull request merged**